themes/digital.gov/src/scss/new/_collection.scss

```diff
@@ -30,6 +28,17 @@
     width: 0.9em;
   }
 
+  .icon {
+    @include u-margin-top("1px");
+    @include u-margin-right(1);
+    img {
+      @include u-width(2);
+      @include u-maxw(2);
+      @include u-position("relative");
+      @include u-top("2px");
+    }
+  }
+
   ul {
```

Comment:
For maintainability let's try to style classes wherever possible and stick to 3 levels deep for nesting.

Reply:
I've added a comment for future refactoring since most of these fixes apply to the legacy code.